.github/workflows/build-and-test.yml

```diff
@@ -74,7 +74,7 @@ jobs:
                   XUNIT_PATH: .\tests\ImageSharp.Web.Tests # Required for xunit
 
             - name: Update Codecov
-              uses: codecov/codecov-action@v1.0.7
+              uses: codecov/codecov-action@v1
```

Comment:
We'll have to do this in all our repos. They've changed things which causes a 400